MapleServer2/AIScripts/PrisonGovernor2Battle.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Maple2Storage.Types;
+using MapleServer2.Data.Static;
+using MapleServer2.Managers.Actors;
+using MapleServer2.Types;
+
+namespace MapleServer2.AIScripts;
+
+public class PrisonGovernor2Battle : AIState
+{
+    public override void OnEnter(Npc npc)
+    {
```

Comment:
In the same sense that a field has Triggers and TriggerLoader, I feel as though we should do the same for NPCs, so we wouldn't have to feed in the NPC instance to get NPC variables and values.